LICENSES/BSD-2-Clause.txt

```diff
@@ -0,0 +1,9 @@
+Copyright (c) <year> <owner> 
```

Comment:
Curious as to why do we need LICENSES folder with Apache license text and clauses from BSD, CC and MIT? Does LICENSE file not suffice?